src/azure-cli/azure/cli/command_modules/acs/_polling.py

```diff
@@ -0,0 +1,40 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from typing import Optional
+
+from azure.core.polling.base_polling import LocationPolling, _is_empty, BadResponse, _as_json
+
+
+class RunCommandLocationPolling(LocationPolling):
+    """Extends LocationPolling but uses the body content instead of the status code for the status"""
```

Comment:
May I ask why you want to inherit your own Poller and use this special status logic?

Reply:
Basically, I want the message in the progress bar to be dynamically generated based on the poller HTTP response. `LocationPolling` basically checks the status code and returns a hardcoded value of "InProgress" if HTTP status 201/202 hence why we have to extend the class. 

BodyContentPolling only supports PUT/PATCH methods which is not applicable here.

Reply:
Git it, thanks~